app/Http/Controllers/Traits/IsAdminIndex.php

```diff
@@ -33,7 +33,7 @@ protected function isolateAuthorizedFrameworks(QueryBuilder $query, string $tabl
             $query->where(function ($query) use ($tableName, $frameworkNames, $user) {
                 foreach ($frameworkNames as $framework) {
                     $frameworkPermission = 'framework-' . $framework;
-                    if ($user->hasPermissionTo($frameworkPermission)) {
```

Comment:
`can` and `hasPermissionTo` appear to consistently test the same thing, but `can` doesn't simply returns false instead of throwing an error when the Permission hasn't been defined.